### PR TITLE
Feature/ttnctl components

### PR DESCRIPTION
**Pull request by romeovs**
Add these commands to ttnctl:
- `ttnctl components list`
- `ttnctl components add`
- `ttnctl components info`
- `ttnctl components token`

### CODE REVIEW

**Comment by htdvisser** [comment]
Please make these commands hidden, as most `ttnctl` users won't need them

File: ttnctl/cmd/components_info.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	"fmt"
+
+	"github.com/TheThingsNetwork/ttn/ttnctl/util"
+	"github.com/spf13/cobra"
+)
+
+var componentsInfoCmd = &cobra.Command{
+	Use:   "info [type] [ComponentID]",
+	Short: "Get information about a network component.",
+	Long:  `components info can be used to retreive information about a network component.`,
```
Comment (johanstokking) [inline ttnctl/cmd/components_info.go]: retrieve

File: ttnctl/cmd/components_token.go [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+	"fmt"
+
+	"github.com/TheThingsNetwork/ttn/ttnctl/util"
+	"github.com/spf13/cobra"
+)
+
+var componentsTokenCmd = &cobra.Command{
+	Use:   "token [type] [ComponentID]",
+	Short: "Get the token for a network component.",
+	Long:  `components token gets a singed token for the component.`,
```
Comment (johanstokking) [inline ttnctl/cmd/components_token.go]: signed

File: ttnctl/cmd/components_token.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+import (
+	"fmt"
+
+	"github.com/TheThingsNetwork/ttn/ttnctl/util"
+	"github.com/spf13/cobra"
+)
+
+var componentsTokenCmd = &cobra.Command{
+	Use:   "token [type] [ComponentID]",
```
Comment (johanstokking) [inline ttnctl/cmd/components_token.go]: Type

File: ttnctl/cmd/components_token.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+		if len(args) != 2 {
+			cmd.UsageFunc()(cmd)
+			return
+		}
+
+		account := util.GetAccount(ctx)
+
+		token, err := account.ComponentToken(args[0], args[1])
+		if err != nil {
+			ctx.WithError(err).WithField("type", args[0]).WithField("id", args[1]).Fatal("Could get component token")
```
Comment (johanstokking) [inline ttnctl/cmd/components_token.go]: Could not

**Comment by htdvisser** [comment]
Fixed Travis builds in `refactor`.

**Pull request merged**